pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -490,6 +490,69 @@ var testsRequiringHistoryRewrite = map[testCoordinates]string{
 		jobName:  "periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-upgrade",
 		testName: "[sig-api-machinery] Kubernetes APIs remain available with reused connections",
 	}: "history correction on improperly loose disruption criteria, expires 2022-01-15",
+
+	{
+		jobName:  "aggregated-azure-ovn-upgrade-4.10-micro-release-openshift-release-analysis-aggregator",
```

Comment:
you need to wire against the periodic that is the basis for this aggregated job.

Reply:
> you need to wire against the periodic that is the basis for this aggregated job.

you can click on one of the sub-job-runs to get that name.

Reply:
ok, I'm taking the actual periodic jobName from the name of the 10 jobs in `aggregation-jobrun-summary.html/FinshedJobs` output -- hopefully this is correct.